PoGo.Necrobot.Window/Model/PlayerInfoModel.cs

```diff
@@ -1,20 +1,51 @@
-﻿using POGOProtos.Enums;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using PoGo.NecroBot.Logic.Event;
+using System.Threading.Tasks;
+using TinyIoC;
 using POGOProtos.Networking.Responses;
 using POGOProtos.Inventory;
+using POGOProtos.Enums;
+using PoGo.NecroBot.Logic.Event;
 using PoGo.NecroBot.Logic.Utils;
+using PoGo.NecroBot.Logic.State;
+using POGOProtos.Inventory.Item;
+using System.Timers;
 
 namespace PoGo.Necrobot.Window.Model
 {
     public class PlayerInfoModel : ViewModelBase
     {
+        private DateTime expires = new DateTime(0);
+        private System.Timers.Timer tmr = new Timer()
```

Comment:
In Line 13 System.Timers is already defined, so i don't see it necessary to be referenced twice